Installation.md

```diff
@@ -0,0 +1,259 @@
+## Setup ##
+
+### Notices before installation ###
+
+We assume you use [RVM](https://rvm.io/). If you don't, then replace `rvmsudo` with `sudo` during the installation process.
+
+:exclamation: RVM 1.17.x may have some [problems with rvmsudo](http://stackoverflow.com/questions/13765520/rvmsudo-command-not-working-properly).
+
+An X server is currently needed for browser tests to pass when using Selenium. `Xvfb` will do, but remember to set your `DISPLAY`.
```

Comment:
Well - this is outdated and has been for a while; since the switch to phantomjs we no longer need to have X for the tests, right?